src/languages/de.json

```diff
@@ -0,0 +1,101 @@
+{
+    "chat": {
+        "actions": {
+            "banUser": "Ban Benutzer",
```

Comment:
Isn't that exactly what happens when you 'make a user a moderator' ? I might get these make thingies on the wrong end